ui/src/app/pages/za-settings/modals/create-car-form/create-car-form.component.ts

```diff
@@ -67,19 +75,20 @@ export class CreateCarFormComponent implements OnInit {
     private ref: DynamicDialogRef,
     private config: DynamicDialogConfig,
     private fb: FormBuilder,
-
-    private carService: CarService,
-  ) { }
+    private sessionService: SessionService,
+    private carService: CarService
+  ) {}
 
   ngOnInit(): void {
     this.car = this.config.data.car;
     this.readOnly = this.config.data.readOnly;
-
+    this.isAdminRole = this.sessionService.isRealAdminOrHigher;
     let worksheet: ICarForm | null;
 
     try {
-      const worksheetSource = FieldsUtils.getFieldValue(this.car, FieldNames.Car.worksheet) || '';
-      worksheet = JSON.parse(worksheetSource)
+      const worksheetSource =
+        FieldsUtils.getFieldValue(this.car, FieldNames.Car.worksheet) || '';
```

Comment:
No need tabulation here

const worksheetSource = FieldsUtils.getFieldValue(this.car, FieldNames.Car.worksheet) || '';